doc/run_experiment.rst

```diff
@@ -142,6 +142,13 @@ possible settings for each section is provided below, but to summarize:
     You also can optionally use predetermined folds with the
     :ref:`cv_folds_file <cv_folds_file>` setting.
 
+    .. note::
+
+        Note that when using classifiation, SKLL will automatically reduce the
```

Comment:
Nitpick:  No need for the "Note that" phrase since this appears in a `Note` box.

![](https://www.dropbox.com/s/op3wauzq88mryxe/Screenshot%202017-08-22%2014.09.47.png?dl=1)